recipes/email2pdf/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "email2pdf" %}
+{% set version = "0.9.9.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

Comment:
It seems like you uploaded the PyPI version but aren't the upstream maintainer, is this correct?

Reply:
So the issue is that the original maintainer of `email2pdf` archived the package, now I forked it and updated it. But I agree that it might be confusing for users. So I decided to rename the fork to `email2pdf2` to have a clear difference.